### ISSUE TITLE
roketdizi.co: breakage

### ISSUE THREAD

**Issue by punisher018** [uBlockOrigin/uAssets#23976]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://roketdizi.co/dizi/interview-with-the-vampire/sezon-2/bolum-4`

### Category

breakage

### Description

Because of that filter `##div[id^="gpt_ad_"]` the page is showing almost empty.

### Other extensions used

-

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![Ekran Alıntısı](https://github.com/uBlockOrigin/uAssets/assets/51531027/9cb39550-c520-46e1-9bfe-201584f42326)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.58.0
Chromium: 125
filterset (summary):
 network: 157503
 cosmetic: 166459
 scriptlet: 36363
 html: 0
listset (total-discarded, last-updated):
 removed:
  user-filters: null
 added:
  https://easylist-downloads.adblockplus.org/cntblock.txt: 1704-0, now
  https://raw.githubusercontent.com/Spam404/lists/master/adblock-list.txt: 7253-0, now
  adguard-cookies: 29874-63, now
  adguard-other-annoyances: 14411-345, now
  adguard-popup-overlays: 27116-419, now
  easylist-annoyances: 4658-16, now
  easylist-notifications: 3020-82, now
  fanboy-cookiemonster: 51105-3965, now
  fanboy-thirdparty_social: 68-0, now
  ublock-annoyances: 5914-24, now
  ublock-cookies-adguard: 1173-0, now
  ublock-quick-fixes: 87-0, now
 default:
  TUR-0: 13235-13, now
  easylist: 86579-650, now
  easyprivacy: 51011-1028, now
  plowe-0: 3627-8, now
  ublock-badware: 9153-5, now
  ublock-filters: 38338-608, now
  ublock-privacy: 928-8, now
  ublock-unbreak: 2327-6, now
  urlhaus-1: 16140-0, now
filterset (user): [array of 1936 redacted]
trustedset:
 added: [array of 18 redacted]
 removed:
  chrome-extension-scheme
switchRuleset:
 added: [array of 4 redacted]
hostRuleset:
 removed:
  behind-the-scene * * noop
  behind-the-scene * image noop
  behind-the-scene * 3p noop
  behind-the-scene * inline-script noop
  behind-the-scene * 1p-script noop
  behind-the-scene * 3p-script noop
  behind-the-scene * 3p-frame noop
userSettings:
 advancedUserEnabled: true
 cloudStorageEnabled: true
 userFiltersTrusted: true
hiddenSettings:
 trustedListPrefixes: ublock- user-
supportStats:
 allReadyAfter: 945 ms (selfie)
 maxAssetCacheWait: 424 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 8
 network:
  adform.net: 3
  diken.xyz: 1
  googletagmanager.com: 3
  storj.cloud: 1
 extended:
  ###mPlayerFd
  ##a[href^="https://cutt.ly/"]
  ##div[id^="gpt_ad_"]
  ##+js(abort-on-property-read, initPop)
```
</details>

**Comment by stephenhawk8054** [comment]
```adb
roketdizi.co#@#div[id^="gpt_ad_"]
roketdizi.co##div[id^="gpt_ad_"]:not(#gpt_ad_HEADER_FULL_BANNER)
roketdizi.co##.ad
||images.macellan.online/allads/$image
```

cc @Yuki2718 For Adguard Turkish

**Comment by Yuki2718** [comment]
I can't reproduce.

**Comment by punisher018** [comment]
Now it doesn't reproduced for me either. Yesterday, even when I opened the same site from my phone's internet, it was showing the same site incorrectly. But the day before that, when I opened it from the PC, it showed broken, when I opened it from the phone, it showed it properly. (desktop view) 

I don't know if it depends on the internet service provider. I tried it in different browsers on the same day on the same internet service provider and the result was the same. (so it's not a cookie problem etc.)

@stephenhawk8054 Have you ever reproduce it?
